src/score.js

```diff
@@ -0,0 +1,19 @@
+class Score {
+
+  constructor() {
+    this.frameScore = 0;
+    this.totalScore = 0;
+  }
+
+  currentFrame(frame_rolls) {
+    this.frameScore = (frame_rolls.roll_one + frame_rolls.roll_two);
+  }
+
+  scoreSoFar(frames_played) {
```

Comment:
this seems so much cleaner than my scoring method, didn't know about reduce, could you perhaps talk me through the calls as it seems almost recursive.